src/app/components/Footer/Link/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
+import { useState, useEffect, MouseEvent } from 'react';
+import onClient from '#lib/utilities/onClient';
+import { FooterLink } from '#app/models/types/serviceConfig';
+import styles from './index.styles';
+
+interface LinkProps extends FooterLink {
+  inline?: boolean;
+  onClick?: (e: MouseEvent<HTMLAnchorElement>) => void;
+  onlyShowIfJSenabled?: boolean;
+}
+
+export default ({
+  text,
+  href,
+  inline = false,
+  lang,
+  onClick,
+  onlyShowIfJSenabled = false,
+}: LinkProps) => {
+  const [isVisible, setVisible] = useState(onlyShowIfJSenabled !== true);
+
+  useEffect(() => {
+    if (onlyShowIfJSenabled && onClient()) {
```

Comment:
Probably just from the direct port of the non-TS version, but I think `onClient` can be removed here. `useEffect` should only ever run on the client-side, so this part of the check will always be `true`.

Reply:
Leaving as is for the moment as not sure what will happen when removed, so won't do it for this piece of work.

We have removed the onClient part though